modules/customeredit.php

```diff
@@ -217,7 +217,7 @@
                 }
             }
 
-            if ($customerdata['ten'] != '') {
+            if ($customerdata['ten'] != '' && $customerdata['ten'] != $LMS->GetCustomerTen($_GET['id'])) {
```

Comment:
Raz małą raz dużą. Dlaczego?

Reply:
To omyłka - poprawiłem https://github.com/chilek/lms/pull/2259/commits/67b4d85c901026bbccddb4b6e45075d9ad1c766d. 

W sumie to dobre pytanie dlaczego część metod od małej część od dużej. 
Może należałoby w 28.x to też usystematyzować?

Reply:
> To omyłka 

Sława Ukrainie!

Reply:
Powinniśmy trzymać się standardowej konwencji `camelCase`?

Reply:
Poczytaj o `camelCase`.

Reply:
Zauważyłem nieścisłość w LMS, poczytałem trochę o tym - dla PHP zaleca się `UpperCamelCase` albo `camelCase`.
Zadecyduj i poprawimy hurtem wszędzie w 28.x.

Reply:
Poprawiłem nazwę metody w kodzie, wrzucisz czy coś jeszcze znalazłeś?

Reply:
> Zauważyłem nieścisłość w LMS, poczytałem trochę o tym - dla PHP zaleca się `UpperCamelCase` albo `camelCase`.
> Zadecyduj jak ma być i poprawimy hurtem wszędzie w 28.x?

Na ten moment po prostu wszystko, co dodajemy do kodu konstruujemy zgodnie z regułami `camelCase`.
Nawet phpcs pod to przygotowaliśmy swego czasu:
https://github.com/chilek/lms-plus/blob/a30d25438c2a79d6a939e4b967396f95d757ed6f/phpcs3.xml#L16